doc/cartridge_kubernetes_guide/index.rst

```diff
@@ -0,0 +1,1016 @@
+
+3. **helm** is a package manager for Kubernetes apps. We need the
+   version 3.3.x. Installation instructions can be found
+   `here <https://helm.sh/docs/intro/install/>`_.
+
+   .. code-block:: console
+
+      $ helm version
+      ---
+      version.BuildInfo{Version:"v3.3.1", GitCommit:"249e5215cde0c3fa72e27eb7a30e8d55c9696144", GitTreeState:"clean", GoVersion:"go1.14.7"}
+
+4. **minikube** is a tool for creating a local Kubernetes cluster. We
+   need the version 1.12 or higher. Installation instructions can be found
+   `here <https://kubernetes.io/ru/docs/tasks/tools/install-minikube/>`_.
+
+   .. code-block:: console
+
+      $ minikube version
+      ---
+      minikube version: v1.12.3
+      commit: 2243b4b97c131e3244c5f014faedca0d846599f5-dirty
+
+5. **kind** (optional) is another tool for creating a local cluster. It
+   can be used instead of the minicube. Installation instructions can be
+   found
+   `here <https://kind.sigs.k8s.io/docs/user/quick-start/#installation>`_.
+
+   .. code-block:: console
+
+      $ kind version
+      ---
+      kind v0.9.0 go1.15.2 linux/amd64
+
+.. _cartridge_kubernetes_creating_an_application:
+
+--------------------------------------------------------------------------------
+Creating application
+--------------------------------------------------------------------------------
+
+Let's create a Cartridge application named ``test-app`` using ``cartridge-cli``:
+
+.. code-block:: console
+
+   $ cartridge create --name test-app
+   ---
+      • Create application test-app
+      • Generate application files
+      • Initialize application git repository
+      • Application "test-app" created successfully
+
+In the ``test-app`` directory, we get the app created from a template:
+
+.. code-block:: console
+
+   $ ls test-app
+   ---
+   ...
+
+   instances.yaml
+   test-app-scm-1.rockspec
+   ...
+
+The app is fully functional and can respond to the HTTP GET request ``/hello``.
+
+.. NOTE::
+
+   Check the cartridge version in *test-app-scm-1.rockspec*:
+
+   .. code:: lua
+
+      dependencies = {
+          ...
+          'cartridge == 2.3.0-1',
+          ...
+      }
+
+The version of Cartridge must be **>= 2.3.0**. Starting from this version,
+Cartridge waits for an instance to become available on its DNS address during
+the instance start. This is required for correct operations in Kubernetes. For
+versions below 2.3.0, an application must be customized independently.
+See the
+`example <https://github.com/tarantool/tarantool-operator/blob/master/examples/kv/key-value-store/init.lua#L27-L71>`_
+of how to do this.
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Building application
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Let's create a docker image using ``cartridge-cli``:
+
+.. code-block:: console
+
+   $ cartridge pack docker --tag vanyarock01/test-app:0.1.0-0-g68f6117
```

Comment:
Where does the `g68f6117` part come from? Is this a shortened hash of the current git commit?